sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server-obo/README.md

```diff
@@ -97,10 +97,10 @@ azure:
     client-secret: [resource-server-application-client-secret]
     tenant-id: [tenant-id-registered-by-application]
     app-id-uri: api://sample-client-id
-    authorization:
+    web-api-clients:
```

Comment:
Let's use `authorization` instead, we won't use two sets of properties for webapi and webapp seperately.